actionpack/lib/action_controller/metal/params_wrapper.rb

```diff
@@ -253,7 +253,8 @@ def process_action(*args)
         # This will display the wrapped hash in the log file.
         request.filtered_parameters.merge! wrapped_filtered_hash
       end
-      super
+    ensure
+      return super
```

Comment:
Maybe clarify the behaviour here with a comment explaining that it will rescue all exceptions - it's not a construct that you see every day.

Otherwise 👍